libs/langserve/langserve/server.py

```diff
@@ -1,31 +1,31 @@
+from inspect import isclass
 from typing import (
     Any,
     AsyncIterator,
     Dict,
     List,
+    Literal,
     Mapping,
     Sequence,
     Type,
     Union,
 )
 
-from langchain.load.dump import dumpd, dumps
-from langchain.load.load import load
 from langchain.schema.runnable import Runnable
 from typing_extensions import Annotated
 
 try:
     from pydantic.v1 import BaseModel
 except ImportError:
-    from pydantic import BaseModel
+    from pydantic import BaseModel, create_model
```

Comment:
Shouldn't this use `langchain.pydantic_v1` ?

Reply:
It feels a odd to do this given that langserve is a separate package. Do you see any benefits to doing this? Do things become easier if langchain starts supporting pydantic 2 code (i.e., without the v1 namespace)